msm/src/prover.rs

```diff
@@ -135,14 +137,22 @@ where
     // Adding MVLookup
     if let Some(ref lookup_env) = lookup_env {
         // -- first m(X)
-        polynomials.push((
-            DensePolynomialOrEvaluations::DensePolynomial(&lookup_env.lookup_counters_poly_d1),
-            None,
-            PolyComm {
-                unshifted: vec![G::ScalarField::zero()],
-                shifted: None,
-            },
-        ));
+        polynomials.extend(
```

Comment:
Shouldn't there be a corresponding change in the verifier too?

Reply:
Hidden in the iterator implementation.